media/css/mozorg/internet-health/health-subpage.scss

```diff
@@ -0,0 +1,747 @@
+.theme-inclusion {
+    .main-header {
+        background-image: url('/media/img/mozorg/internet-health/inclusion/inclusion-header-bg.png');
+    }
+
+    .head-pagenav a:before {
+        background-image: url('/media/img/mozorg/internet-health/inclusion/inclusion-nav-sprite.png');
+    }
+}
+
+//* -------------------------------------------------------------------------- */
+// Common elements
+body {
+    border-top: none;
+}
+
+#outer-wrapper {
+    background: #fff;
+}
+
+main .content {
+    max-width: $width-desktop;
+}
+
+
+//*--------------------------------------------------------------------------*/
+// Page masthead
+#masthead {
+    background: #fff;
+    color: #000;
+    padding: 0;
+    width: 100%;
+
+    .masthead-logo {
+        font-size: 100%;
+        height: 30px;
+        left: 0;
+        margin: 0;
+        padding: 10px 0 0;
+        position: absolute;
+        text-align: center;
+        top: 0;
+        width: 100%;
+
+        img {
+            height: 30px;
+        }
+    }
+
+    .download-button {
+        display: none;
+
+        .download-list {
+            margin-bottom: 0;
+        }
+
+        .download-link {
+            @include font-size(14px);
+            border-radius: 0;
+            border: none;
+            padding: 15px 20px;
+            text-transform: uppercase;
+        }
+    }
+
+    .fx-privacy-link {
+        display: none;
+    }
+
+    .masthead-nav-main {
+        .toggle {
+            background-color: #000;
+        }
+
+        .nav-main-menu a:hover {
+            border-color: #000;
+            color: #000;
+        }
+    }
+
+    @media #{$mq-tablet} {
+        .masthead-nav-main {
+            margin-right: 0;
+
+            .nav-main-menu {
+                padding: 15px 0;
+
+                a:link,
+                a:visited {
+                    color: #000;
+                }
+
+                li {
+                    border-left: 1px solid #000;
+
+                    &:first-child {
+                        border-left: none;
+                    }
+                }
+            }
+        }
+
+        .masthead-logo {
+            float: left;
+            padding: 10px 0;
+            position: relative;
+            width: auto;
+        }
+
+        .download-button {
+            display: block;
+            float: right;
+
+            .download-link {
+                padding: 15px 12px;
+            }
+        }
+    }
+
+    @media #{$mq-desktop} {
+        .download-button .download-link {
+            padding: 15px 20px;
+        }
+    }
+}
+
+html[dir="rtl"] {
+    @media #{$mq-tablet} {
+        #masthead .masthead-nav-main .nav-main-menu li {
+            border-left: none;
+            border-right: 1px solid #fff;
+
+            &:first-child {
+                border-right: none;
+            }
+        }
+    }
+}
+
+
+//*--------------------------------------------------------------------------*/
+// Health subpages navigation
```

Comment:
We'll need a better plan for what the navbar looks like on mobile. For now this is good enough with only two links, but once we add three and four it's gonna get clunky.